pygaggle/run/evaluate_passage_reader.py

```diff
@@ -99,52 +93,48 @@ def main():
     args = apb.parser.parse_args()
     options = PassageReadingEvaluationOptions(**vars(args))
 
-    logging.info("Loading Reader Model and Tokenizer.")
+    logging.info("Loading the Retrieval File")
+    with open(options.retrieval_file) as f:
+        data = json.load(f)
+
+    logging.info("Evaluating Topk Accuracies")
+    nPassages = len(list(data.values())[0]["contexts"])
+    for k in TOP_K_ACCURACY:
```

Comment:
this part seems not clean.
Instead of calling the script multiple times here, I think it will be cleaner and faster to modify the evaluation script directly. and call the script only once.
i.e. let the evaluation gives multiple topk accuracies in one pass

Reply:
Agreed, will be cleaner if this is done!